data/alpha/register/social-housing-provider.yaml

```diff
@@ -1,6 +1,9 @@
 fields:
 - social-housing-provider
 - name
+- organisation
```

Comment:
we added organization field to the register to reference where the entry appears in another register